crates/namada/src/ledger/vp_host_fns.rs

```diff
@@ -50,9 +52,9 @@ pub type EnvResult<T> = std::result::Result<T, RuntimeError>;
 
 /// Add a gas cost incured in a validity predicate
 pub fn add_gas(
-    gas_meter: &mut VpGasMeter,
+    gas_meter: &mut impl DerefMut<Target = VpGasMeter>,
```

Comment:
Do we need the `&mut` when requesting `DerefMut`?

Reply:
Good q - we have the same `RefCell` type in native and wasm env, so I'll change these to just pass it around

Reply:
done in 0c98d6d7d30d79ef959ebfa6ce020b9053122fc5